### PR TITLE
Admin Router: adding clarifications of limitations to Readme

### PR DESCRIPTION
**Pull request by DominikDary**
## High-level description

Admin Router: adding clarifications of limitations to Readme

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

- [DCOS-42624](https://jira.mesosphere.com/browse/DCOS-42624) Master Admin Router responds 404 on /service/jenkinsX.

## Related tickets (optional)

Other tickets related to this change:

  - [DCOS_OSS-<number>](https://jira.mesosphere.com/browse/DCOS_OSS-<number>) Foo the Bar so it stops Bazzing.

## Checklist for all PRs

  - [ ] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and review process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using GitHub's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by DominikDary** [comment]
@mesosphere-mergebot bump-ee

File: packages/adminrouter/extra/src/README.md [hunk 1]
```diff
@@ -431,6 +432,9 @@ frameworks that:
 * become disconnected from Mesos for some reason, but *MAY* come back at some
   point
 
+Note that in DC/OS Enterprise  only frameworks that are started by Root
```
Comment (adamtheturtle) [inline packages/adminrouter/extra/src/README.md]: There is a double space after 'Enterprise'. I think that this is meant to be ", ".

Comment (DominikDary) [reply packages/adminrouter/extra/src/README.md]: fixed

File: packages/adminrouter/extra/src/README.md [hunk 2]
```diff
@@ -312,17 +312,18 @@ order to support the `/service` URL path prefix clients are using.
...
 In the case of Mesos frameworks, this becomes a bit more complex - all the
 frameworks are visible in Mesos' `/state-summary` endpoint output so in theory
 it is possible to access a framework launched using Marathon-on-Marathon or by
-hand.  Unfortunately, Mesos as of now does not enforce framework name
-uniqueness and thus it is possible to launch multiple frameworks with the same
-name. AR in such case will route in a non-deterministic manner. Root Marathon
-by default enforces unique names for all the tasks and frameworks it has under
-control.
+hand (except in DC/OS Enterprise where only frameworks that are started by Root 
+Marathon can be routed to).  Unfortunately, Mesos as of now does not enforce
```
Comment (vespian) [inline packages/adminrouter/extra/src/README.md]: > except in DC/OS Enterprise where only frameworks that are started by Root Marathon can be routed to).
Doesn't it apply to OSS as well? Also, I hope that you will not mind that I throw my 2 cents here and ask if we can clarify where this limitation stems from (i.e. mesos roles).

Comment (jgehrcke) [reply packages/adminrouter/extra/src/README.md]: Pawel in DC/OS Enterprise Admin Router sees the filtered state-summary, whereas in in DC/OS it sees the complete state-summary.

**Comment by ahairape** [comment]
@adamtheturtle I think, the current document is what a customer is supposed to read. The component documentation page links to it: https://docs.mesosphere.com/1.11/overview/architecture/components/#admin-router
https://docs.mesosphere.com/1.12/overview/architecture/components/

**Comment by jgehrcke** [comment]
Who is finding and issuing override commands?

**Comment by adamtheturtle** [comment]
@jgehrcke we discussed in today’s stand up that this would be a good opportunity for Dominik to go through the process.

**Comment by jgehrcke** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/azure/hybrid-cloud https://jira.mesosphere.com/browse/DCOS-42798

**Comment by jgehrcke** [comment]
The override commands did not take effect, sadly.

**Comment by orsenthil** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/azure/hybrid-cloud  DCOS-42798

**Comment by orsenthil** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/azure/arm DCOS-41675

**Comment by orsenthil** [comment]
@mesosphere-mergebot label ship-it

**Pull request merged**